docs/modules/ROOT/pages/runbooks/mariadbgalera/MySQLGaleraClusterSmall.adoc

```diff
@@ -0,0 +1,53 @@
+= Alert rule: MySQLGaleraClusterSmall
+
+== icon:glasses[] Overview
+
+This alert indicates that the Galera cluster has less than 3 nodes
+
+== icon:bug[] Steps for Debugging
+
+See xref:how-tos/mariadbgalera/debug.adoc[How to debug MariaDB Galera] on how to determine the cluster status and show the Logs of MariaDB.
+
+=== One Pod in CrashLoopBackoff
+
+Sometimes it may happen that data of one of the nodes becomes corrupted.
+
+If, and ONLY IF `mariadb_state` shows that the remaining two nodes are synced, operational and have the same state_uuid's, clear the data dir of the broken pod.
+
+Since you can't access the broken pod while it's in CrashLoopBackoff, you have to access the volume via the node:
```

Comment:
An alternative would be to delete the PVC and pod of the broken node -> faster and no figuring out where data is on the underlying machine.

Reply:
Good point! I guess the process would be like this?

- kubectl delete pvc
- kubectl delete pod

And done?

Reply:
Yes, that's it :)